packages/babel-helpers/src/helpers/usingCtx.js

```diff
@@ -0,0 +1,78 @@
+/* @minVersion 7.23.0 */
+
+export default function _usingCtxFactory() {
```

Comment:
Could you wrap the old helper rather than the new one, so that the new one remains smaller?